ziti/cmd/ziti/cmd/api/json.go

```diff
@@ -89,3 +110,62 @@ func (self *GabsWrapper) StringSlice(path string) []string {
 	}
 	return nil
 }
+
+type Gabs2Wrapper struct {
```

Comment:
An aside from this PR's goal is:

I want to reduce the usage of gabs with generated structures. Gabs is great when you don't know the JSON format, but if it is a contract-based object, I'm against using it as it creates scenarios where things can break and no one notices till an end-user reports the issue.

Reply:
Makes sense. For the next pass I'll see about converting to using the generated rest client